### DISCUSSION TITLE
What is format of the customization spec ?

### DISCUSSION THREAD

**Discussion by lugerlee** [vmware/govmomi#3248] [Community Help] [answered]
What is format of the customization spec NAME as bellow?
Appreciate in advance for your help!

```
Usage: govc vm.customize [OPTIONS] [NAME]

Customize VM.

Optionally specify a customization spec NAME.
```

**Comment by dougm** [comment, marked as answer]
The name used to create the spec, in the UI for example: https://docs.vmware.com/en/VMware-vSphere/7.0/com.vmware.vsphere.vm_admin.doc/GUID-EB5F090E-723C-4470-B640-50B35D1EC016.html

**Comment by lugerlee** [comment]
Many thanks!
But I want to reset `JoinWorkgroup`  when clone a Windows VM.
I can modify the exported spec file(in XML), but `govc import.spec` does not reconize the xml file.
